Лабораторные работы

Mihanakis opened the pull request.

Mihanakis committed:
Add Lab3

Mihanakis committed:
Add Tic-Tac-Toe Game

Mihanakis committed:
Add Tic-Tac-Toe Game

EgorOb reviewed (approved):
Хорошо. В по игре направление верное. Только попробуйте сильно не привязываться к размеру 3х3. мы эту задачу делаем как обучение абстракции. Понятно что если знаешь конкретный объём данных, то можно оптимизировать под него всё, но сейчас основная задача попробовать обобщить случаи, чтобы не писать отдельную функцию 4х4, 5х5 и т.д.

Mihanakis committed:
Add Tic-Tac-Toe Game

Mihanakis committed:
Изменил функцию для вывода квадратных матриц.

Mihanakis committed:
Убрал "магические числа" в задании условий.

Mihanakis committed:
Добавлена функция превращения списка в матрицу.
Закончена функция проверки на ошибки.
В работе функция вычислений внутри матрицы в общем виде.

Mihanakis committed:
Добавлена функция превращения списка в матрицу.
Закончена функция проверки на ошибки.
В работе функция вычислений внутри матрицы в общем виде.

Mihanakis committed:
Добавлен ввод для расчёта квадратных матриц любого размера. Добавлен ввод и проверка ввода размера предстоящей матрицы.

EgorOb reviewed (approved):
Хорошо

Mihanakis committed:
Убрана подцветка строк с ошибкой, т.к. её не видно в терминале.

Mihanakis committed:
Add Lab4

Mihanakis committed:
Заменился файл, возвращаю обратно.

EgorOb reviewed (approved):
Зачтено

Mihanakis committed:
Исправлена ошибка:
При победе на последнем шаге выдавалось сообщение о ничьей, а не о победе одного из игроков.

Mihanakis committed:
Переписано в виде List comprehension

Mihanakis committed:
Убраны 2 лишние строки

Mihanakis committed:
Исправлено замечание

Mihanakis committed:
Добавляю лабораторную 1 из курса PY110

EgorOb reviewed (approved):
Зачтено
